app/controllers/organizations_controller.rb

```diff
@@ -71,6 +72,18 @@ def destroy
     end
   end
 
+  def remove_user
+    if current_organization.users.count < 2
```

Comment:
Should we be enforcing this at the model level?

Reply:
We can add a new method called `one_owner_remains?` to organization 👌